Taskfile.yml

```diff
@@ -1003,14 +1016,25 @@ tasks:
       # Wait for the other operator to be ready - not using controller:wait-for-operator-ready because that also waits for CRDs, etc which this doesn't install
       - "kubectl wait --for=condition=ready --timeout=2m pod -n tenant1-system -l control-plane=controller-manager"
 
+  controller:kind-create-helm:
```

Comment:
Ah! this wasn't supposed to be there. I think some merge conflicts might've added it back. Since I made some changes, and it was removed in another PR. 
PS. I've removed it now. Thanks for pointing out.